### ISSUE TITLE
Zoom (or high DPI) causes weird artifacts in the splitter

### ISSUE THREAD

**Issue by jnm2** [KirillOsenkov/SourceBrowser#164]
100% zoom looks fine (80% zoom for me at 125% dpi):

![image](https://user-images.githubusercontent.com/8040367/115131476-6e6e5e80-9fc6-11eb-839e-fcd09ca0c295.png)

90% at 125% dpi:

![image](https://user-images.githubusercontent.com/8040367/115131482-82b25b80-9fc6-11eb-8d15-b7cfdd77f823.png)

100% at 125% dpi:

![image](https://user-images.githubusercontent.com/8040367/115131489-8cd45a00-9fc6-11eb-93d1-65818173a750.png)

110% at 125% dpi:

![image](https://user-images.githubusercontent.com/8040367/115131499-98c01c00-9fc6-11eb-8de2-f7392dfecbae.png)

**Comment by KirillOsenkov** [comment]
I suspect these might be browser issues with support for frames

**Comment by KirillOsenkov** [comment]
Which browser is this? Which OS? have you tried other browsers or OSs?

**Comment by jnm2** [comment]
Firefox 87.0 x64 on Windows 10 20H2.

Good thought. Chrome and Edge don't show anything at all:

![image](https://user-images.githubusercontent.com/8040367/115131778-a676a100-9fc8-11eb-901c-78f073b86ec8.png)

Until you hold the mouse button and start moving, then Chrome and Edge look like this:

![image](https://user-images.githubusercontent.com/8040367/115131792-cad27d80-9fc8-11eb-857b-861392e5069f.png)

**Comment by KirillOsenkov** [comment]
Yeah I've seen these types of issues before. Browsers just don't support this well. Unfortunately getting rid of frames would mean a full rewrite and I really don't have any time.
